common/culture/innovations/wc_regional_innovations.txt

```diff
@@ -44,3 +51,106 @@ innovation_war_camels = {
 	flag = global_regional
 	flag = tribal_era_regional
 }
+###Tech
+##Tribal
+
+innovation_sea_faring = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	region = custom_sea_locked
+	icon = @sea_faring
+	
+	potential = {
+		NOT = {
+			this.culture_group = culture_group:vrykul_group
+		}
+	}
+	modifier = {
+		embarkation_cost_mult = -0.8
+		garrison_size = -0.2
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regional
+}
+innovation_kyparite_sap = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	region = custom_kyparite_regions
+	icon = @kyparite_sap
+
+	modifier = {
+		garrison_size = 0.05
+		skirmishers_damage_mult = 0.1
+		skirmishers_toughness_mult = 0.05
+		monthly_prestige_gain_mult = 0.05		
+	}
+
+	flag = global_regional
+	flag = tribal_era_regional
+}
+##Early Medieval
+innovation_frigid_warfare = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	region = custom_frigid_regions
+	icon = @taiga
+
+	modifier = {
+		taiga_advantage = 3
+		mountains_advantage = 2
+		taiga_attrition_mult = -0.2
+		mountains_attrition_mult = -0.2
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regional
+}
+innovation_arid_warfare = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	region = custom_arid_regions
+	icon = @desert
+
+	modifier = {
+		desert_advantage = 3
+		drylands_advantage = 2
+		desert_attrition_mult = -0.2
+		drylands_attrition_mult = -0.2
```

Comment:
There are 2 more terrains associated with arid: `oasis` and `desert_mountains`.

Reply:
Are these two provinces in the game as of right now?

Reply:
> Are these two terrains in the game as of right now?

Yes.